.github/CONTRIBUTING.md

```diff
@@ -1,64 +1,65 @@
+Often is a bug you encounter fixed in a newer release.
 
-Examples of behavior that contributes to creating a positive environment
-include:
+You can optionally also try out development builds to see, if your bug is fixed in those.
 
-* Using welcoming and inclusive language
-* Being respectful of differing viewpoints and experiences
-* Gracefully accepting constructive criticism
-* Focusing on what is best for the community
-* Showing empathy towards other community members
+The same rules apply for when making a feature request.
 
-Examples of unacceptable behavior by participants include:
+### No duplicate issue
+Make sure that there isn't any existing issue, which is still open, or is closed with a solution/explanation.  
+Opening a separate issue for a bug report or feature request, that already exists on the issue tracker only slows down the process of fixing the bug or implementing the feature.
 
-* The use of sexualized language or imagery and unwelcome sexual attention or advances
-* Trolling, insulting/derogatory comments, and personal or political attacks
-* Public or private harassment
-* Publishing others' private information, such as a physical or electronic address, without explicit permission
-* Other conduct which could reasonably be considered inappropriate in a professional setting
+If an issue with the bug or feature you want to report/suggest exists, comment on it with your info (bug reports) or give it a :thumbsup: (Feature Request) to show that this is important for you.
 
-### Our Responsibilities
+### Issue isn't caused by external source
+PlaceholderAPI provides a feature to have extensions (separate jar files) for placeholders. This gives it a possability that an issue you encounter is caused by said extensions or a separate plugin that uses those extensions.  
+In those cases should you report the issue to the issue tracker of the extension or plugin.
 
-Project maintainers are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instances of unacceptable behavior.
+## Pull requests
+As an open source project are we welcoming all contributions to improve PlaceholderAPI, being it changes to its code like bug fixes or new features, or contributions to its documentation such as the Wiki or the Javadoc.
 
-Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
+### Code contributions
+Any contributions to PlaceholderAPI's code should be done towards the `develop` branch. Targeting the `master` branch in your Pull request may get it closed without warning.  
+Additionally should you follow the `Google Codestyle` when changing the code and Javadoc.
```

Comment:
No, this isn't the case. PAPI uses it's own codestyle, which is in the config folder iirc.

Reply:
It uses a codestyle that is adabted from the Google Codestyle, which @extendedclip once suggested to be added (And caused this huge debate with Sxtanna)